dataPreservation.js

```diff
@@ -0,0 +1,26 @@
+const userName = document.getElementById('name');
+const userEmail = document.getElementById('email');
+const userMessage = document.getElementById('message');
+const form = document.getElementById('contact-form');
+const contactFormData = {
+  fullName: '',
+  email: '',
+  message: '',
+};
+
+form.addEventListener('input', () => {
+  contactFormData.fullName = userName.value;
+  contactFormData.email = userEmail.value;
+  contactFormData.message = userMessage.value;
+  localStorage.setItem('contactFormData', JSON.stringify(contactFormData));
+});
+
+window.onload = () => {
+  const contactData = JSON.parse(localStorage.getItem('contactFormData'));
+
```

Comment:
Good job writing this logic, but for some reasons, the data are not writing to locasl storage. I recomend fixing this issue in order to make fully satisfy the project requirement.

HINT: Consider listening for the `submit` event, instead of `input`

Here is a video of the behavior i am seeing.

[screen-capture.webm](https://github.com/melkamu12/Portfolio-DesktopVersionM/assets/84930355/3326213b-b2f3-43c3-b46c-a55927b91336)